build/charts/theia/values.yaml

```diff
@@ -297,3 +299,37 @@ theiaManager:
     tlsMinVersion: ""
   # -- Log verbosity switch for Theia Manager.
   logVerbosity: 0
+zookeeper:
```

Comment:
I'm thinking about putting this section under installZookeeper instead of as an individual section, as this section should only be useful when user enable installZookeeper.

Reply:
I've moved it under installZookeeper. Thanks.